### DISCUSSION TITLE
Dagster-related Job Openings

### DISCUSSION THREAD

**Discussion by molliemarie** [dagster-io/dagster#7214] [Ideas]
# What is this?
This discussion is for companies looking for people with Dagster experience. Both permanent positions and contract work are welcome.

# Companies
Please use the following format followed by a brief description of the role:

COMPANY_NAME | ROLE | LOCATION | Full-time | $  | LINK

Please only post if you personally are part of the hiring company—no recruiting firms or job boards. Only one post per company.

Good luck with recruiting!

# Our openings
We're going to start this out by highlighting some current jobs open at Elementl (the folks who make Dagster!)

## Developer Advocate
Elementl | Developer Advocate | US Remote | Full-time | https://boards.greenhouse.io/elementl/jobs/5035087003

✨ Develop product education materials
✨ Represent Dagster in the developer community
✨ Engage with users
✨ Influence product direction on behalf of the community

As the developer advocate, you will create product education materials that will help our users to smoothly onboard to Dagster and succeed in learning about the project and best practices, adapting Dagster to their custom infrastructure, evaluate Dagster for their organization, or anything in between. You will also be one of the early members of the dev rel team and will have the opportunity to help us shape the team as it grows.

## Other openings

- Elementl | Technical Writer | US Remote | Full-time | https://boards.greenhouse.io/elementl/jobs/4331034003
- Elementl | Community Manager | US Remote | Full-time | https://boards.greenhouse.io/elementl/jobs/5035349003
- Elementl | Engineering Manager | US Remote | Full-time | https://boards.greenhouse.io/elementl/jobs/5022361003
- Elementl | Software Engineer | US Remote | Full-time | https://boards.greenhouse.io/elementl/jobs/4203324003
- Elementl | Sales Engineer | US Remote | Full-time | https://boards.greenhouse.io/elementl/jobs/4974027003
- Elementl | Head of Product | US Remote | Full-time | https://boards.greenhouse.io/elementl/jobs/4203357003
- Elementl | Software Engineer: Intern | US Remote | Full-time | https://boards.greenhouse.io/elementl/jobs/4205182003

**Comment by zyd14** [comment]
[Empirico](https://www.empiricotx.com/) is looking for another Data Infrastructure Engineer! We're looking for engineering-focused data engineers and data-focused software engineers who want to build tooling and infrastructure for our team of data engineers, bioinformaticians, and scientists to enable them to perform petabyte-scale analyses on the largest genomic and phenotypic datasets in the world in search of novel therapeutics.  We've recently adopted Dagster as a core component of our cross-platform workflow management services.  We also heavily leverage Spark, Databricks, and AWS services. If any of that sounds interesting to you, please apply [here](https://www.empiricotx.com/software-engineer-data-infrastructure/)!

**Comment by AndreaGiardini** [comment]
Hey folks :wave: 
I am a freelance Infrastructure Engineer / DevOps based in the NL and I've been hanging around the Dagster community for quite a while now, especially in the #dagster-kubernetes channel on Slack.

My experience with cloud is mainly in GCP / AWS . I have been working for quite some time with Kubernetes and I am a certified CKA / CKAD / CKS. In my career, I've been working for a mix of startups and big organizations. During the past year or so, I have been helping a AI/ML startup set up their infrastructure and automating their stack (mainly working with Terraform, CI/CD, Kubernetes, GCP, Helm, Jupyterhub, Cloud Run, Serverless).

I am always looking for new opportunities to work with Dagster in my projects.

Website: https://andreagiardini.com/
LinkedIn: https://www.linkedin.com/in/giardiniandrea/
Dagster talk at PyCon Italy 2022: https://www.youtube.com/watch?v=MIhF6Fh0AXw

Feel free to send me an email or connect on LinkedIn

**Comment by geoHeil** [comment]
https://www.magenta.at/ is looking for a Senior Cloud Data Architect and Lead Data Engineer to join us:

- https://www.magenta.at/karriere/start?jh=lv781fir9hg1yh1eq2a6g8wkc5js72e
- https://www.magenta.at/karriere/start?jh=gntsi7pqbxlm1npfp2b9khhhkibbfuq

We are using Dagster & DBT as a core component for our new cloud native data stack

**Comment by maxmansouri** [comment]
Hello there, not sure how far this will go as this discussion seems outdated, but worth a shot! I am launching a data analytics as a service company, providing end to end data solutions for clients. I'm looking for a developer experienced in Dagster and DBT to orchestrate ETL jobs in and out of postgresql. Going after clients with small-mid size data. Looking to expand down the line into something more robust and sustainable, taking on bigger clients. If you have flexibility and ambition to tackle some projects please reach out to me maxmansouri@outlook.com. Thank you.
